Analysis/src/SignalObjectSelector.cpp

```diff
@@ -294,8 +285,10 @@ bool SignalObjectSelector::PassHH_legacy_LeptonSelection(const LepCandidate& lep
     if(!(lepton->PassedOldDecayMode())) return false;
     if(!lepton->Passed(e_id.first, e_id.second)) return false;
     if(!lepton->Passed(mu_id.first, mu_id.second)) return false;
-    if(legId == 1 && !lepton->Passed(TauIdDiscriminator::byIsolationMVArun2017v2DBoldDMwLT2017,DiscriminatorWP::Medium)) return false;
-    if(legId == 2 && !lepton->Passed(TauIdDiscriminator::byIsolationMVArun2017v2DBoldDMwLT2017,DiscriminatorWP::VVLoose)) return false;
+    if(legId == 1 && !lepton->Passed(GetTauVSjetDiscriminator().at(mode).first,
+                                     GetTauVSjetDiscriminator().at(mode).second)) return false;
+    if(legId == 2 && !lepton->Passed(GetTauVSjetDiscriminator().at(mode).first,
+                                     GetTauVSjetDiscriminator().at(mode).second)) return false;
```

Comment:
for the second leg you should ask to pass lower sideband WP, not the signal region WP